freqtrade/freqtradebot.py

```diff
@@ -590,6 +595,7 @@ def _notify_buy(self, trade: Trade, order_type: str) -> None:
         msg = {
             'trade_id': trade.id,
             'type': RPCMessageType.BUY,
+            'buy_tag': trade.buy_tag,
```

Comment:
The addition of this should also be added to `webhook-config.md` - to document for which hooks this is available (it will be available implicitly by adding it here, just noone will know about it).

Obviously applies to all messages that have this field added ...